python/llm/test/benchmark/check_results.py

```diff
@@ -20,26 +20,44 @@
 import sys
 import argparse
 import pandas as pd
+from omegaconf import OmegaConf
 
 def main():
     parser = argparse.ArgumentParser(description="check if the number of lines in html meets expectation")
-    parser.add_argument("-n", "--expected_lines", type=int, dest="expected_lines",
-                        help="the number of expected html lines", default=0)
-    parser.add_argument("-f", "--folder_path", type=str, dest="folder_path",
-                        help="The directory which stores the .csv files", default="./")
+    parser.add_argument("-c", "--csv_name", type=str, dest="csv_name",
+                        help="name of csv", default="")
+    parser.add_argument("-y", "--yaml_name", type=str, dest="yaml_name",
+                        help="name of yaml", default="")
     args = parser.parse_args()
 
-    csv_files = []
-    for file_name in os.listdir(args.folder_path):
-        file_path = os.path.join(args.folder_path, file_name)
-        if os.path.isfile(file_path) and file_name.endswith(".csv"):
-            csv_files.append(file_path)
-    csv_files.sort()
-    
-    number_of_expected_lines=args.expected_lines
-    num_rows = len(pd.read_csv(csv_files[0], index_col=0))
-    if num_rows!=number_of_expected_lines:
-        raise ValueError("The number of arc perf test results does not match the expected value. Please check carefully.")
+    special_word_in_csv = args.csv_name
```

Comment:
why special_word_in_csv? why not directly input the csv name (test1.csv)?

Reply:
Because the `test_api` and `date` is dynamic. For arc perf test, we use `sed -i 's/{today}/{today}_test1/g' run.py` to rename csv.